src/components/shared/inputs/textField.scss

```diff
@@ -8,4 +10,14 @@
 
 .input {
   padding: 5px;
+  background-color: $background;
```

Comment:
This isn't a good idea without changing the color css as well since you can't see the default black text in dark mode.

Honestly we're probably gonna want to update this in it's own PR cause this component needs work. 

We could do it in this PR, but I think it's already getting scope creepy. We might just want to revert this components css to what it was for now.

Reply:
i agree but the default text color in dark mode is white